doc/classes/InputEventKey.xml

```diff
@@ -65,6 +65,7 @@
 		</member>
 		<member name="key_label" type="int" setter="set_key_label" getter="get_key_label" enum="Key" default="0">
 			Represents the localized label printed on the key in the current keyboard layout, which corresponds to one of the [enum Key] constants or any valid Unicode character.
+			Key labels are intended for use in key prompts.
```

Comment:
Similar sentiment to a prior comment.
```suggestion
			Represents the localized label printed on the key in the current keyboard layout, which corresponds to one of the [enum Key] constants or any valid Unicode character. Key labels are meant for key prompts.
```
It is admittedly weird that `keycode` in this PR provides an example but `key_label` does not.

Reply:
It can be used for any displayed prompt, so not sure if there's any specific example.